packages/create-cloudflare/src/templates.ts

```diff
@@ -203,10 +205,27 @@ export const selectTemplate = async (args: Partial<C3Args>) => {
 		question: "What would you like to start with?",
 		label: "category",
 		options: [
-			{ label: "Hello World example", value: "hello-world" },
-			{ label: "Framework Starter", value: "web-framework" },
-			{ label: "Demo application", value: "demo" },
-			{ label: "Template from a Github repo", value: "remote-template" },
+			{
+				label: "Hello World example",
+				value: "hello-world",
+				description:
+					"[Recommended for beginners] Build an empty starter to configure project structure and bare-bones",
+			},
+			{
+				label: "Framework Starter",
+				value: "web-framework",
+				description: "Create a website with a JavaScript framework",
+			},
+			{
+				label: "Demo application",
+				value: "demo",
+				description: "Build a web application using Cloudflare Workers",
```

Comment:
`Build a web application using Cloudflare Workers` -- not all these examples here are web apps, and this doesn't sufficiently distinguish this category from the `hello world` templates. Id probably highlight that the examples here are more full-fledged and showcase advanced usage of the developer products.